databooks/common.py

```diff
@@ -21,6 +24,14 @@ def get_logger(name: str, level: str = "INFO") -> logging.Logger:
     return logging.getLogger(name)
 
 
+def set_verbose(logger: logging.Logger) -> None:
+    verbose_level = logging.DEBUG
+    logger.setLevel(verbose_level)
+    logger.debug(
+        f"verbose mode: setting log level to {logging.getLevelName(verbose_level)}"
```

Comment:
My OCD kicks in: could you capitalize the first letter? 😅

Reply:
Affirmative. BTW, this comes from go, where people start complaining when I use a capitals to start log/error messages 😅 .